File: 100-Days-Series/Day 92/SearchCharacterInString.js

// Function to find Character in a string
const getFindCharacter=(str,ch)=>{
    for(let i=0;i<str.length;i++){
        if(str.charAt(i)==ch){
            return i;
        }
    }
    return -1;
}
// Let's Start Here
const str="Veer Singh Lodhi";
const ch='m';
res=getFindCharacter(str,ch);
if(res!=-1)
    console.log("Found at Index "+res);
else
    console.log("Not Found!!");

File: 100-Days-Series/Day 92/SearchCharacterInString.py

# Function to find character in a string
def get_find_character(str,ch):
    for i in range(0,len(str)):
        if str[i]==ch:
            return i
    return -1

# Let's Start Here
str=input('Enter a string : ')
ch=input('Enter Character to find : ')
res=get_find_character(str,ch)
if res!=-1:
    print("Found at Index ",res)
else:
    print("Not Found!")